tests/e2e/specs/pacbio/pacbio_run_create.cy.js

```diff
@@ -106,4 +148,52 @@ describe('Pacbio Run Create view', () => {
     // Ensure we made the request
     cy.wait('@deleteRun')
   })
+
+  // need to work out why it can't find binding kit box barcode
+  it.skip('allows for the selection of well defaults', () => {
```

Comment:
Skipped test, is this still the case?

Reply:
Yes. It may well have been failing due to the input bug. I am planning to go back to this as part of the techincal debt story DPL-433-3